docs/src/data/hid.js

```diff
@@ -5182,7 +5182,7 @@ export default [
     ],
     documentation: "https://usb.org/sites/default/files/hut1_2.pdf#page=134",
     os: {
-      windows: null,
+      windows: false,
```

Comment:
```suggestion
      windows: true,
```
This also seems to work for me, tested on Windows 11 desktop (shows the brightness slider but doesn't affect the external display) and Windows 10 Surface Book (actually sets screen brightness down as expected.

Reply:
Looks good, committing changes. An aside, what version of windows is the Surface Book on? My testing was done on Win 10 1909.

Reply:
Thanks! This is Surface Book Gen 1, on Windows 10 21H2 19044.1415.